packages/talos_certifier_adapters/src/kafka/config.rs

```diff
@@ -1,49 +1,57 @@
 use rdkafka::ClientConfig;
+use std::collections::HashMap;
 use talos_certifier::env_var;
 
 #[derive(Debug)]
-pub struct Config {
+pub struct KafkaConfig<'a> {
     pub brokers: Vec<String>,
-    pub topic_prefix: String,
-    pub consumer_topic: String,
-    pub producer_topic: String,
+    pub topic: String,
     pub client_id: String,
     pub group_id: String,
     pub username: String,
     pub password: String,
+    pub producer_config_overrides: HashMap<&'a str, &'a str>,
```

Comment:
Why not use `String` for key and value, rather than doing the above? Because now the compiler isn't sure of the lifetime of those key/value and we are forced to explicitly set the lifetime. 

But at the same time, if you look at `talos_certifier_adapters/src/certifier_kafka_pg.rs`, we are explicitly setting the ` 'a ` on the root function. i.e we are saying we expect these to live through the entire life of the root function. Therefore why not just use `Hashmap<&'static str, &'static str>`?

Reply:
In this case we might as well use static lifetimes. But as a general practice i think we should try to use more reference rather than owned types.

Reply:
Or a `Cow` with a static lifetime. I'd have no beef with that approach.